src/Illuminate/Http/Client/RequestException.php

```diff
@@ -13,6 +13,13 @@ class RequestException extends HttpClientException
      */
     public $response;
 
+    /**
+     * The truncation length for the exception message.
+     *
+     * @var int|false
+     */
+    public static $truncateAt = 120;
```

Comment:
```suggestion
     * @var int
     */
    public static int $truncateAt = 120;
```

Reply:
`truncateAt` can be `false`, and the other property in this class doesn't have a type, so I'm going to leave it up to the Laravel team to decide on this.